.github/workflows/dockerhub.yml

```diff
@@ -3,9 +3,11 @@ name: Docker Hub build
 on:
   push:
```

Comment:
Only added this because I checked dockerhub and there is not repository for autoware.auto in the stol org

Reply:
we don't create image for autoware.auto btw. It gets checked out like carma-msgs